.github/workflows/sims.yml

```diff
@@ -105,22 +55,10 @@ jobs:
       - uses: actions/checkout@v2.4.0
       - uses: actions/setup-go@v2.1.5
         with:
-          go-version: 1.17
-      - uses: technote-space/get-diff-action@v5.0.2
-        with:
-          PATTERNS: |
-            **/**.go
-            go.mod
-            go.sum
-      - uses: actions/cache@v2.1.7
-        with:
-          path: ~/go/bin
-          key: ${{ runner.os }}-go-runsim-binary
-        if: "env.GIT_DIFF != ''"
```

Comment:
lets keep the scope of this pr small. Can we spin these changes out into another PR?

at least the removal of things, the bump in go version is needed